src/test/java/com/github/coleb1911/ghost2/commands/modules/fun/ModuleCalculatorTest.java

```diff
@@ -0,0 +1,83 @@
+package com.github.coleb1911.ghost2.commands.modules.fun;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.coleb1911.ghost2.commands.meta.CommandContext;
+import com.github.coleb1911.ghost2.commands.meta.Module;
+import com.github.coleb1911.ghost2.commands.meta.ModuleInfo;
+import com.github.coleb1911.ghost2.commands.meta.ReflectiveAccess;
+import com.github.coleb1911.ghost2.utility.RestUtils;
+import discord4j.core.object.util.Permission;
+import discord4j.core.object.util.PermissionSet;
+import org.jsoup.Jsoup;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+/**
+ * @author chicago4550
+ */
+
+
+
+@Service
+public final class ModuleCalculatorTest extends Module {
```

Comment:
Thank you for the quick reply. Now that I am running it again today in IntelliJ instead of atom and PowerShell, I am getting errors on MacAdapter.class not being able to be in a top-level directory. Where is MacAdapter.class, or is that an issue with how I have everything configured on my end? In addition, runtimeBuilder cannot find my OpenJDK Java.exe, despite it pointing directly to it. Any ideas on how to fix these two issues?